src/lib/es2015.iterable.d.ts

```diff
@@ -214,10 +214,6 @@ interface PromiseConstructor {
     race<T>(values: Iterable<T | PromiseLike<T>>): Promise<T>;
 }
 
-declare namespace Reflect {
-    function enumerate(target: object): IterableIterator<any>;
-}

```

Comment:
The&nbsp;`enumerate(…)`&nbsp;method was&nbsp;removed in&nbsp;**ES2016** from&nbsp;both&nbsp;`Reflect` and&nbsp;`ProxyHandler`s, due&nbsp;to&nbsp;implementers deciding&nbsp;against&nbsp;it.

Reply:
Extracted to: <https://github.com/microsoft/TypeScript/pull/38967>